components/mindmap/menu-items/edit.js

```diff
@@ -170,6 +172,18 @@ const PopperCard = ({ el, poppers }) => {
                 onChange={getChangeHandler(setContent)}
               />
             </FormGroup>
+            <FormGroup>
+              <Label for="audio">Audio (URL)</Label>
+              <Input
+                type="url"
+                name="audio"
+                id="audio"
+                value={audio}
+                maxLength="2,048"
```

Comment:
```suggestion
                maxLength="2048"
```

The numeric value with a comma gets truncated to 2 when parsed.